solutions/swb-reference/src/SWBStack.ts

```diff
@@ -477,13 +463,13 @@ export class SWBStack extends Stack {
     return bucket;
   }
 
-  private _createSecureS3Bucket(s3BucketId: string, s3OutputId: string, mainAcctEncryptionKey: Key): Bucket {
+  private _createSecureS3Bucket(s3BucketId: string, s3OutputId: string, encryptionKey: Key): Bucket {
```

Comment:
We have [constructs](https://github.com/aws-solutions/solution-spark-on-aws/tree/develop/workbench-core/infrastructure/src) defined for workbenchDynamodb and workbenchSecureS3Bucket as well. Please look into using these constructs.

Reply:
acknowledged. Will not refactor for the scope of this ticket and PR.